etip/restful_api/serializers.py

```diff
@@ -23,10 +23,21 @@ class Meta:
           'is_in_exodus',
           'code_signature',
           'network_signature',
+          'api_key_ids',
           'website',
           'maven_repository',
           'group_id',
           'artifact_id',
           'gradle',
+          'created',
+          'updated',
+          'creation_date',
+          'capability',
+          'advertising',
+          'analytic',
+          'network',
+          'comments',
```

Comment:
What's the concern?  Are you worried about bandwidth usage?  My guess is that this API will only ever be lightly used anyway, since hardly anyone has a key.  Including all fields makes syncing between other sources possible.